make/photon/db/postgresql/initial-notaryserver.sql

```diff
@@ -0,0 +1,4 @@
+CREATE DATABASE notaryserver;
+CREATE USER server;
+alter user server with encrypted password 'password';
```

Comment:
So we are hard code notary password to "password" . It it possible we share the same password with harbor db?  which is also random generated.

Reply:
as notary is using no password way to connect DB, add the password here is because that pgsql raises one exception on "no password to connect db".  I guess that pqsql could not accept one db  both hold connect with and without password.

But, I agree that we could provide better way to generate password random. it could be fixed in another PR.